app/page.tsx

```diff
@@ -163,20 +165,18 @@ const CreateProjectModal: React.FC<{
   )
 }
 
-export default function Home() {
+const  Home = observer(() => {
   const {
-    projects,
-    fetchProjects,
     createProject,
   } = useProjectListContext();
 
+  const projects = ModelStore.projects;
```

Comment:
If a component is listening for updates, you can use the MobX properties as variables or even as dependencies in useEffects and mobx will force re-render accordingly.